tests/transformers/test_transformers_model_export.py

```diff
@@ -3924,3 +3632,65 @@ def test_text_generation_task_chat_serve(text_generation_pipeline):
         and output_dict["usage"]["completion_tokens"] < 10
     )
     assert output_dict["usage"]["prompt_tokens"] < 20
+
+
+HF_COMMIT_HASH_PATTERN = re.compile(r"[a-z0-9]{40}")
```

Comment:
```suggestion
HF_COMMIT_HASH_PATTERN = re.compile(r"^[a-z0-9]{40}$")
```

or replace `match` with `fullmatch`.